torchci/rockset/metrics/__sql/force_merge_red_avg.sql

```diff
@@ -0,0 +1,22 @@
+WITH all_merges AS (
+  SELECT
+    skip_mandatory_checks,
+    LENGTH(failed_checks) AS failed_checks_count,
+  FROM
+    commons.merges
+  WHERE
+    _event_time >= PARSE_DATETIME_ISO8601(:startTime)
+    AND _event_time < PARSE_DATETIME_ISO8601(:stopTime)
+    AND owner = :owner
+    AND project = :project
```

Comment:
Is "project" the same as "repository" here? Want to make sure we're not accidentally pulling data from other repos in this query

Reply:
Yeah, `project` and `repository` are interchangeable as GitHub term AFAIK